collector/collector.go

```diff
@@ -691,6 +668,17 @@ func (c *collector) CollectMetrics(mts []plugin.Metric) ([]plugin.Metric, error)
 		return nil, fmt.Errorf("No metrics found")
 	}
 
+	// add labels as tags to metrics
+
+	for i := range metrics {
+		rid := metrics[i].Namespace[2].Value
+		if rid != "root" {
+			// add labels only for docker cointainer, skip the host
+			metrics[i].Tags = c.containers[rid].Specification.Labels
```

Comment:
I don't see how it's better when we don't have any other tags, it only introduces (small) overhead both in terms of code lines (copy or allocate map) and performance.